src/BenchmarkDotNet/Helpers/DisposeAtProcessTermination.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace BenchmarkDotNet.Helpers
+{
+    /// <summary>
+    /// Ensures that Dispose is called at termination of the Process.
+    /// </summary>
+    public class DisposeAtProcessTermination : IDisposable
```

Comment:
```suggestion
    internal abstract class DisposeAtProcessTermination : IDisposable
```

Reply:
I hope you are OK with the class being public:

..\src\BenchmarkDotNet.Diagnostics.Windows\EtwDiagnoser.cs(19,27,19,39): error CS0060: Inconsistent accessibility: base class 'DisposeAtProcessTermination' is less accessible than class 'EtwDiagnoser\<TStats>'

Reply:
Oh I missed that those other classes are public. In that case, it's fine.